megamek/src/megamek/server/ServerHelper.java

```diff
@@ -395,4 +395,22 @@ else if ((entity.heat >= 14) && !entity.isShutDown()) {
             vPhaseReport.addAll(s.destroyEntity(entity, "pilot death", true));
         }
     }
+    
+    /**
+     * Helper function that causes an entity to sink to the bottom of the water
+     * hex it's currently in.
+     */
+    public static void sinkToBottom(Entity entity) {
+        IHex fallHex = entity.getGame().getBoard().getHex(entity.getPosition());
+        int waterDepth = 0;
+        
+        // we're going hull down, we still sink to the bottom if appropriate
+        if (fallHex.containsTerrain(Terrains.WATER)) {
+            // *Only* use this if there actually is water in the hex, otherwise
+            // we get ITerrain.LEVEL_NONE, i.e. Integer.minValue...
+            waterDepth = fallHex.terrainLevel(Terrains.WATER);
+        }
+        
+        entity.setElevation(-waterDepth);
```

Comment:
Is this always appropriate? Should they always be set to elevation 0 if not in water, or should this just be skipped?

Reply:
I think I better limit it to just in water, and also it occurs to me I forgot to check for the entity being on top of a bridge.